src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/UserController.kt

```diff
@@ -24,17 +26,64 @@ import org.springframework.web.bind.annotation.ResponseStatus
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.externalusersapi.config.ErrorResponse
 import uk.gov.justice.digital.hmpps.externalusersapi.repository.UserFilter.Status
+import uk.gov.justice.digital.hmpps.externalusersapi.repository.entity.User
 import uk.gov.justice.digital.hmpps.externalusersapi.service.UserSearchService
 import uk.gov.justice.digital.hmpps.externalusersapi.service.UserService
 import java.time.LocalDateTime
 import java.util.UUID
 
-@Validated
 @RestController
 @RequestMapping("/users")
 @Tag(name = "/users", description = "External User Controller")
 class UserController(private val userSearchService: UserSearchService, private val userService: UserService) {
 
+  @GetMapping
+  @Operation(
+    summary = "Search for a user.",
+    description = "Search for a user."
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "OK.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = UserDto::class)
+          )
+        ]
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      ),
+      ApiResponse(
```

Comment:
instead of 404  , add 204 as per method return type

Reply:
Done